modules/backend/formwidgets/markdowneditor/assets/js/markdowneditor.js

```diff
@@ -156,7 +156,7 @@
     }
 
     MarkdownEditor.prototype.onResize = function() {
-        this.editor.resize()
```

Comment:
@ayumihamsaki everything inside of this file as well :)

Reply:
yah was thinking that as well on that one